community/how_to_ask_on_discourse.md

```diff
@@ -1,4 +1,128 @@
+
+**Self-contained**
+
+Anyone should be able to immediately run your example code.  This means that your example should include all import statements.  Your example should include any custom functions that are necessary.  If you code relies on data that is loaded from an external file/site, consider creating or generating toy data in your example itself:
+  - `my_data = [[2, 7], [31, 9]]`
+  - `mydata = np.random.default_rng(12345).random((2,2))`
+
+If there are calls that (implicitly) rely on random seeds, explicitly set the seed
+  - [`np.random.default_rng(12345)`](https://numpy.org/doc/stable/reference/random/generator.html)
+  - `pm.sample(random_seed=12345)`
+
+**Relevant output**
+
+This includes error messages, warnings, diagnostics, statistics, and/or plots.  Describe what about these results are relevant to your question.  What are you seeing that you **did not** expect?  What are you **not** seeing that you **did** expect?
+
+**Your example should demonstrate your problem**
+
+After streamlining your code down to just the essentials, make sure that you double-check that your example code still produces your problem.  If not, perhaps you accidentally solved your own problem!
+
+
+(title)=
+## A Good Title
+The title of your topic is critical because it's the first thing people will see when you submit your question.  If the question doesn't seem relevant or interesting or if people can't understand what your question might be about, then you may not get people to read and answer your question.
+
+Highlight what you believe to be the critical aspects of your question.  Finally, if you have trouble summarizing what aspects of your problem are important, you can try waiting until you have written your entire topic, including the {ref}`mwe` before trying to write the title.
+
+**Good titles**:
+- "Why doesn't pm.sample() use all my cores?"
+- "Multiprocessing BrokenPipeError: [Errno 32] Broken pipe"
+- "How I get az.plot_dist() to show priors?"
+
+**Not so good titles**:
+- "How do I do Bayesian inference?"
+- "Is there a better way to implement this model?"
+- "Hierarchical models"
+
+
+(versions)=
+## Versions
+Please include information about what versions of the relevant packages you have installed and how they were installed, including:
+
+- PyMC version
+- Aesara version
+- Python version
+- System, etc. (e.g., linux, Mac, Windows, jupyter, docker, Google Colab)
+- How you installed PyMC (e.g., pip, conda, mamba)
+
+
+
+## Tags <a id='tags'></a>
+Discourse includes two ways to classify topics: categories and tags.  Each topic belongs to exactly one category.  The most popular category is `Questions`, but `Development` and `Sharing` are also quite popular.  Each topic may have more than one tag.  When you post your topic, search through the tags for any key words that might be relevant.  Tags include things like `bug`, `gaussian_process`, and `gpu`.
```

Comment:
I would highlight the importance of categories and tags even more. I think it might we worth mentioning for example that unlike GitHub, people can subscribe to specific categories or tags. 

I for example check out discourse from time to time whenever I feel like it, there is no routine, sometimes I check it every day for a while and sometimes I spend a week or more without answering. But I am subscribed to the "arviz" tag, so whenever a new question is posted using this tag I get an email notification, and I do check my email roughly every day. If the question is long or complicated I will bookmark it to answer next time I log in to discourse and have time, but if it is short, chances are I'll answer much faster. If no tag is used, I'll probably get to it next time I log in, but if it has been a longish hiatus of several days, it is possible I miss the topic altogether.

Side note: users can no longer post on "Questions" category, they need to use one of the 3 subcategories, v3, v4 or version-agnostic.

Reply:
Are you thinking needs to be added (or emphasized?)?  It already talks about how people can subscribe to tags and categories and it says that this is the reason that selecting categories/tags is important for getting eyeballs, etc.  Should we just try to emphasize the importance of this step more?

Reply:
Maybe we could put the 2nd paragraph in an [fancy style box](https://jupyterbook.org/en/stable/reference/cheatsheet.html#admonitions).

Or add a "Summary" box at the top/bottom (maybe replacing the index if at the top) with bullet points summarizing the advise on the doc, for example (order now is completely random):

* Avoid the "General" category and use {ref}tags
* Use a clear {ref}title
* If code related, make sure to include a {ref}mwe
* ...

Sorry I am not being very actionable in my comments, I'd like this to pop more because I think it is key for discourse to work properly and especially to scale to more users but I'm not sure how to get there

Reply:
I am a bit wary of adding a "summary" at the beginning because it may discourage people from reading anything else.  But I added it at the end.  See what you think.